### DISCUSSION TITLE
Thunderbird

### DISCUSSION THREAD

**Discussion by MTRNord** [jelmer/xandikos#287] [Q&A] [answered]
Hi,
did anyone try yet thunderbird? I tried the /dav prefix suggested in the example in the documentation and thunderbird seems to hit the wellknown files with PROPFIND on the server however it says "no server found" as it seems to not follow the 302. Any ideas on how to fix that? Is that fixable or is that a bug at thunderbird?

**Comment by MTRNord** [comment]
> 19/Feb/2024:10:32:05 +0000] "PROPFIND /.well-known/caldav HTTP/1.1" 302 182 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:115.0) Gecko/20100101 Thunderbird/115.7.0"

is the only request i see when I tell it to discover the server

**Reply by jelmer** [reply]
Yes, I'm using thunderbird regularly and am not seeing any issues. I suspect this is an issue with your setup; how do you have your ingress configured? Where does /.well-known/caldav redirect if you curl it? Does curling /dav/ work?

**Reply by MTRNord** [reply]
Currently it's a traefik without the basic auth (I did tick the thunderbird Box) (I will apply it later today :) I am aware of the implications) using kubernetes. Essentially the example files but the Port corrected to 8080 instead of 8081.

Curl says it does redirect to /dav/ which is also the endpoint in the ingress file. Opening /dav/ in curl obviously yields 404 but I assume that's expected as its just the prefix.

The Well known response is:

```
HTTP/2 302
< alt-svc: h3=":443"; ma=2592000
< content-type: text/plain; charset=utf-8
< date: Mon, 19 Feb 2024 11:02:46 GMT
< location: /dav/
< server: Python/3.11 aiohttp/3.8.5
< strict-transport-security: max-age=63072000; includeSubDomains; preload
< vary: Accept-Encoding
< x-content-type-options: nosniff
< x-xss-protection: 1; mode=block
< content-length: 10
<
* Connection #0 to host (redacted) left intact
302: Found
```

**Reply by MTRNord** [reply]
Ok it seems like /dav/ Lands on a different Server somehow 😵‍💫

**Reply by jelmer** [reply]
I remember Thunderbird had issues accessing CalDAV over HTTP/2, but I was assuming that had been fixed.

**Reply by MTRNord** [reply]
In this case it seems to fail on the ingress side already. using curl with propfind on a caldav endpoint fails too with 404. So likely it speaks to the wrong thing.

**Reply by MTRNord** [reply, marked as answer]
Ok for traefik it seems that the regex is not working. Removing it made it behave.
